chia/server/node_discovery.py

```diff
@@ -38,6 +38,7 @@
 
 class FullNodeDiscovery:
     resolver: Optional[dns.asyncresolver.Resolver]
+    dns_query_ratio = 10  # Contact introducer ("chia dns server") once per dns_query_ratio attempts to get a peer
```

Comment:
this comment is a bit misleading. the "introducer" and "DNS seed" are distinct services. My impression from reading the ticket is that we should:
1. remove the affinity towards finding peers from the DNS seeds (and only fall back onto DNS if we fail to find a peer after a sufficient number of attempts from our peer DB)
2. shuffle the `dns_servers` list in order to pick one at random
3. probably never contact the introducer service (but maybe a safer first step would be to log a warning if we do)

Reply:
Thanks - this exactly what I was misunderstanding. I was aware that we had introducers that handed out peers, but I did not know we had a "DNS seed" service - can you give a brief synopsis of the "DNS Seed" idea?

Reply:
it's the same mechanism bitcoin uses, where the network of caching DNS servers is leveraged to hand out peer IPs.

our root server probably responds with a ranom subset of nodes we know of to each query, and the cache in the network ensures our load stays reasonable. Since we want to encourage community-run DNS seeds, we want to pick which one to use at random.

Reply:
The code seems to call the dns introducer the dns server and the old and busted introducer - just introducer